draft-ietf-httpbis-http2bis.xml

```diff
@@ -1415,6 +1415,16 @@
         <t>
           A list of error codes is included in <xref target="ErrorCodes"/>.
         </t>
+        <t>
+          It is possible that an endpoint will encounter frames that would cause multiple errors. Implementations MAY process
+          multiple errors triggered by a single frame in any order, within the limitations set out in this section. Once an
+          implementation has handled a connection error, no further frame processing will be possible. As a result, connection
+          errors are invariably the last error type handled by an implementation.
+        </t>
+        <t>
+          If an endpoint detects multiple stream errors caused by a single frame, but no connection errors caused by that frame,
+          it MAY choose to handle any one of those errors. If a frame would cause a connection error, that error MUST be handled.
```

Comment:
```suggestion
          it MAY choose to report any one of those errors. If a frame would cause a connection error, that error MUST be reported.
```